### PR TITLE
Make Calendar events clickable

### PR DESCRIPTION
**Pull request by shirlyntang**
Create custom event for calendar in order to add extra functionality (onclick). Added external url link placeholder to utils/events.js so we can specify urls later when we add events.

### CODE REVIEW

File: components/Page_Layout/CustomEvent.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    // This binding is necessary to make `this` work in the callback
+    this.handleClick = this.handleClick.bind(this);
+  }
+  handleClick() {
+  	window.open(this.props.event.url);
+  }
+    render() {
+        return (
+        <h8 onClick={this.handleClick}>
```
Comment (gmundewadi) [inline components/Page_Layout/CustomEvent.js:13]: lets add a onHover css property here to highlight the event

File: utils/events.js [hunk 2]
```diff
@@ -4,13 +4,15 @@ export default [
   {
```
Comment (gmundewadi) [inline utils/events.js:4]: add a link to the Date object documentation at the top of this file for future officers

**Review by gmundewadi — CHANGES_REQUESTED** [review]

**Comment by shirlyntang** [comment]
Added capabilities for text color change on hover. Added Date object documentation link. Also deleted a css file since it wasn't used.

File: components/Page_Layout/CustomEvent.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    this.setState({hover: !this.state.hover});
+    this.render();
+  }
+  handleClick() {
+  	window.open(this.props.event.url);
+  }
+    render() {
+    	var linkStyle;
+	    if (this.state.hover) {
+	      linkStyle = {color: 'black'};
```
Comment (gmundewadi) [inline components/Page_Layout/CustomEvent.js:21]: to match the theme, set the hover color to `rgb(250, 176, 60)`

File: components/Page_Layout/CustomEvent.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	    } else {
+	      linkStyle = {color: 'white'};
+	    }
+
+        return (
+        <h8 style={linkStyle} onMouseEnter={this.toggleHover} onMouseLeave={this.toggleHover} onClick={this.handleClick}>
```
Comment (gmundewadi) [inline components/Page_Layout/CustomEvent.js:27]: is there any way to set the color of the event to `rgb(144, 215, 241)`

**Review by gmundewadi — CHANGES_REQUESTED** [review]
two small color changes for the event. Everything else looks great

**Review by minibapat — APPROVED** [review]

**Review by sriya112 — APPROVED** [review]

**Comment by shirlyntang** [comment]
Keeping white/black colors since it goes better with the calendar events color that I don't think I can change!

**Pull request merged**